Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -651,7 +651,7 @@ void DisplayListPlayerSkia::fill_path_using_paint_style(FillPathUsingPaintStyle
     auto path = to_skia_path(command.path);
     path.offset(command.aa_translation.x(), command.aa_translation.y());
     path.setFillType(to_skia_path_fill_type(command.winding_rule));
-    auto paint = paint_style_to_skia_paint(*command.paint_style, command.bounding_rect().to_type<float>());
+    auto paint = paint_style_to_skia_paint(*command.paint_style, command.path_bounding_rect.to_type<float>());
```

Comment:
The change here seems unrelated to patterns. If it's a separate bug fix, please put it in its own commit.